src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md

```diff
@@ -0,0 +1,106 @@
+# Prometheus Exporter AspNetCore for OpenTelemetry .NET
+
+## Prerequisite
+
+* [Get Prometheus](https://prometheus.io/docs/introduction/first_steps/)
+
+## Steps to enable OpenTelemetry.Exporter.Prometheus.AspNetCore
+
+### Step 1: Install Package
+
+Install
+
+```shell
+dotnet add package OpenTelemetry.Exporter.Prometheus.AspNetCore
+```
+
+### Step 2: Configure OpenTelemetry MeterProvider
+
+* When using OpenTelemetry.Extensions.Hosting package on .NET Core 3.1+:
+
+    ```csharp
+    services.AddOpenTelemetryMetrics(builder =>
+    {
+        builder.AddPrometheusExporter();
+    });
+    ```
+
+* Or configure directly:
+
+    Call the `AddPrometheusExporter` `MeterProviderBuilder` extension to
+    register the Prometheus exporter.
+
+    ```csharp
+    using var meterProvider = Sdk.CreateMeterProviderBuilder()
```

Comment:
would be nicer to show meterProvider being added to DI manually here. Else users who copy this as-is, will let the provider gets GCed!